### ISSUE TITLE
Feature: Multiple representers

### ISSUE THREAD

**Issue by apotonick** [trailblazer/trailblazer#118]
Having discussed many use cases of operation/representers with @danieldraper we came to the following cool additions that _have_ to be in TRB 1.2.
### Multiple Representers

Allow maintaining any number of representers via aliases, in line with how we do it with callback objects.

``` ruby
representer :input do .. end
representer :output do ... end
representer :errors do ... end
```
### Easy inheritance

Allow to automatically inherit from other representers, and provide a directive to include/exclude properties.

``` ruby
representer :errors, inherit: Gemgem::Representer::Errors
representer :errors, inherit: Gemgem::Representer::Errors do 
  # allow fine-tuning, this is already implemented
end
representer :output, inherit: :input, exclude: [:id] # allow :include/:exclude
```

**Comment by spectator** [comment]
😍

**Comment by apotonick** [comment]
Implemented in 2.0.

**Comment by apotonick** [comment]
http://trailblazer.to/gems/operation/2.0/representer.html#full-example
